cmd/antrea-agent/options.go

```diff
@@ -810,3 +817,16 @@ func (o *Options) validateNodePortLocalConfig() error {
 	}
 	return nil
 }
+
+func (o *Options) validateHostNetworkModeOptions() error {
+	ok, defaultHostNetworkMode := config.GetHostNetworkModeFromStr(o.config.HostNetworkMode)
```

Comment:
what justifies the `defaultHostNetworkMode` variable name (specifically the `default` prefix)?